packages/web/src/index.css

```diff
@@ -154,41 +154,47 @@
 
 @layer components {
   .title {
-    font-size: 3rem;
+    font-size: 2rem;
```

Comment:
Not super critical given that we want to focus on apps, but do these affect the notebook look?

Reply:
This changes the title size of the notebook, no impact on the app. I changed this cause the rest of the font are smaller now.